lib/opte/src/engine/port.rs

```diff
@@ -1686,6 +1686,31 @@ impl<N: NetworkImpl> Port<N> {
         }
     }
 
+    fn uft_hit_probe(&self, dir: Direction, ufid: &InnerFlowId, epoch: u64) {
+        cfg_if::cfg_if! {
```

Comment:
Just food for thought here, not blocking. Perhaps we can come up with an abstraction that hides the difference between the kernel probes and USDT, so this sort of logic does not hit everywhere we have probes.

Reply:
Agreed. These are fairly mechanical, so that would be a good fit.